DROD/CharacterDialogWidget.cpp

```diff
@@ -189,6 +190,9 @@ const UINT LIST_LINE_HEIGHT = 22;
 const SURFACECOLOR PaleRed = {255, 192, 192};
 
 std::map<UINT, UINT> CCharacterDialogWidget::speechLengthCache;
+const UINT CCharacterDialogWidget::INDENT_PREFIX_SIZE = 8;
```

Comment:
These constants should be consolidated with the analogous magic numbers featured elsewhere in the code, e.g., CCommandListBoxWidget.

Reply:
@mrimer I'm not quite sure what needs to be consolidated where. These constants appear to be grouped with the other ones in this file, and they don't appear to share a purpose with the ones in CCommandListBoxWidget. That file does have some constants littered about, but they are in the place where they're being used, so it doesn't seem like much of an issue.

Reply:
I appreciate you looking into this.  I possibly misinterpreted the values.  IIRC, to me, it looked like there are identical dimension values in both places, e.g., 8 pixels for an indent size, that possibly need to remain in lockstep in both places for things to work correctly.  If this isn't the case, I'm fine with leaving things as-is for now.